api/presenters/PostPresenter.js

```diff
@@ -13,25 +12,28 @@ var postRelations = (userId, opts) => {
     {responders: qb => qb.column('users.id', 'name', 'avatar_url', 'event_responses.response')},
     (opts && opts.fromProject ? null : {projects: qb => qb.column('projects.id', 'title', 'slug')}),
     (opts && opts.withComments
-      ? {comments: qb => qb.column('comment.id', 'text', 'user_id')}
+      ? {comments: qb => qb.column('comment.id', 'text', 'created_at', 'user_id', 'post_id')}
       : null),
     (opts && opts.withComments
       ? {'comments.user': userColumns}
       : null),
     (opts && opts.withVotes
```

Comment:
since you are setting the default value of `opts` to `{}`, you don't need to do `opts && ...`